.github/workflows/CI.yml

```diff
@@ -35,7 +35,17 @@ jobs:
                 path: ./temp
             
             - name: Copie the files
```

Comment:
Change commit name to smth like: chore: Use -p flag for mkdir and delete old files if neccessary